dh-grpc-proxy/src/discovery_handler.rs

```diff
@@ -51,7 +51,7 @@ impl DiscoveryHandler for DiscoveryHandlerImpl {
         &self,
         request: tonic::Request<DiscoverRequest>,
     ) -> Result<Response<Self::DiscoverStream>, Status> {
-        println!("Connection established!");
+        info!("Connection established!");
         let register_sender = self.register_sender.clone();
         let discover_request = request.get_ref();
         let (mut discovered_devices_sender, discovered_devices_receiver) =
```

Comment:
A few lines below this, why are you setting devices as online within the proxy?:
```
write_input_file(discovery_handler_config);
write_availability_file(ONLINE);
```

Reply:
Nice question, this was used to set a default value for this file and inform the DH about the connection to the agent, specially if running it for the first time but it has been also added to the discovery handler so its redundant here, will be removing from the proxy and leaving it only to the discovery handler as it fits better.

Reply:
Maybe change to fit Akri's DebugEcho DH where assumes online unless "OFFLINE" is written to the file